bower.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "adapt-contrib-matching",
-  "version": "2.0.4",
-  "framework": "^2.0.11",
+  "version": "2.0.5",
```

Comment:
I think we should change the version to 2.1.0 (as we did for the Slider update) to reflect that this is quite a big change